### ISSUE TITLE
[FEATURE REQUEST] WIKI 

### ISSUE THREAD

**Issue by stefan63** [DeeEmm/DIY-Flow-Bench#242]
For a newbee on flow benches maybee a section of the WIKI could help to straighten some ?? in regards of flow bench types :)
Ilustration would be helpfull.
What differs those types and how do they work, where to place the sensors/hoses?
1. MAF based , "the new black"
2. Pressure drop
3. Orifice
4. Venturi

I know how to solder and take messurements (and some coding) but the worings of the flow bench types make my head spin haha...

Kind regards/Stefan

**Comment by DeeEmm** [comment]
Yes we definitely need to expand the existing WIKI info to include the differential style / pressure style bench now that we have it in the code. This is currently missing.

I think it might also be good to leverage @danielerpl's experience with using the PTS plans to build a bench as this is equally applicable to other differential style benches.

**Comment by DeeEmm** [comment]
I'm going to roll this action up with some additional items to improve the on-boarding process for new users.

#239 - Utilise API to assist with initial setup

**Comment by danielerpl** [comment]
> Yes we definitely need to expand the existing WIKI info to include the differential style / pressure style bench now that we have it in the code. This is currently missing.
> 
> I think it might also be good to leverage @danielerpl's experience with using the PTS plans to build a bench as this is equally applicable to other differential style benches.

For sure! if you need any info I am here to help everyone

**Comment by DeeEmm** [comment]
I've now updated the WIKI to include info on the settings, configuration and pins pages. Clicking on the info icon on any tiles on those pages opens up the relevant section of the WIKI in a new window... 

<img width="445" alt="Image" src="https://github.com/user-attachments/assets/d132e453-2cba-4745-b000-7ffacee777c3" />

(maybe we need a 'dark mode' for the GUI 😄 )

<img width="1108" alt="Image" src="https://github.com/user-attachments/assets/ed12145e-ae86-4b1b-8e09-2b52318a7eea" />
